Orange/widgets/data/owfile.py

```diff
@@ -219,6 +247,14 @@ def load_from_file():
                     fn = os.path.join(".", basename)
                     self.information("Loading '{}' from the current directory."
                                      .format(basename))
+            if self.excel_has_sheets(fn):
+                book = open_workbook(fn)
+                if book.nsheets > 1:
```

Comment:
If `excel_has_sheets` returned `True`, then `book.nsheets` is already known to be >1?